### DISCUSSION TITLE
How to make pylsp index whole project?

### DISCUSSION THREAD

**Discussion by PgLoLo** [LazyVim/LazyVim#3102] [Q&A]
Hi! How to make pylsp (LSP for python) to index whole project instead of only opened files? For example, I open large repo, lets say pytorch. I open some python files there, say, `torch.__init__.py`. After that pylsp is initialized, but it index only opened files and nothing else. How to fix it?

**Comment by VersBinarii** [comment]
What is your pylsp config?
